Services.md

```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
+- **/groups/api/v1/invitations/{groupName}/email** : POST : set group invitation email copy
+- **/groups/api/v1/invitations/{groupName}/{userUuid}** : PUT : update an invitation
+- **/groups/api/v1/invitations/{groupName}/{userUuid}** : DELETE : delete an invitation
+- **/groups/api/v1/members/{groupName}** : GET : paginated members
+- **/groups/api/v1/members/{groupName}/{memberUuid}** : DELETE : delete a member
+- **/groups/api/v1/members/{groupName}/{memberUuid}/renew** : POST : renew a member
+- **/groups/api/v1/curators/{groupName}** : POST : add a curator
+- **/groups/api/v1/curators/{groupName}/{memberUuid}/downgrade** : POST : downgrade a curator to a normal member
+- **/groups/api/v1/users** : GET : search for users
+- **/groups/api/v1/users/any** : GET : search for any users
+- **/groups/api/v1/self/{groupName}** : DELETE : leave a group
+- **/groups/api/v1/self/requests** : GET : invitation requests from the logged in user
+- **/groups/api/v1/self/requests/{groupName}** : POST : request to be invited
+- **/groups/api/v1/self/requests/{groupName}** : DELETE : cancel a request to be invited
+- **/groups/api/v1/self/invitations** : GET : invitations for the logged in user
+- **/groups/api/v1/self/invitations/{groupName}** : POST : join a group
+- **/groups/api/v1/self/invitations/{groupName}** : DELETE : reject an invitation to a group
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add a member
+- **/groups/api/v1/sudo/logs/all/raw** : GET : returns an audit log containing records of all actions taken on groups over time
+- **/groups/api/v1/sudo/mail/nda/{userUuid}** : POST : subscribe user to nda mailing list
+- **/groups/api/v1/sudo/mail/nda/{userUuid}** : DELETE : unsubscribe user to nda mailing list
+- **/groups/api/v1/sudo/curators/{groupName}** : GET : curators for a given group
+- **/groups/api/v1/sudo/curators/{groupName}** : POST : add a curator to a given group
+- **/groups/api/v1/sudo/user/{userUuid}** : DELETE : delete a user
+- **/groups/api/v1/sudo/user/inactive** : DELETE : delete inactive users
+- **/groups/api/v1/sudo/user/consolidate** : DELETE :
+- **/groups/api/v1/sudo/user/uuids/members** : GET : all member's UUIDs
+- **/groups/api/v1/sudo/user/uuids/staff** : GET : all staff's UUIDs
+- **/groups/api/v1/sudo/user/data/{userUuid}** : GET : all profile and group data for a given UUID
+- **/groups/api/v1/sudo/user/cis/{userUuid}** : POST : update CIS with a user's current groups
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add member to group
+- **/groups/api/v1/sudo/member/{groupName}/{userUuid}** : DELETE : remove member from group
+- **/groups/api/v1/sudo/trust/groups/{groupName}** : PUT : 
+- **/groups/api/v1/sudo/groups/inactive** : GET : list inactive groups
+- **/groups/api/v1/sudo/groups/inactive/{groupName}** : DELETE : delete inactive group
+- **/groups/api/v1/sudo/groups/reserve/{groupName}** : POST : 
+- **/groups/api/v1/sudo/transfer** : POST :
+
```

Comment:
- **/groups/api/v1/forms/email/** : GET : simple html form to send emails
- **/groups/api/v1/forms/email/bcc** : POST : send email on behalf of DinoPark
- **/groups/api/v1/forms/email/group** : POST : send email to a group